src/scalper.ts

```diff
@@ -154,18 +151,11 @@ async function scalperPerp() {
     }
 
   // GAMMA SCALPING //
-  // Calc gamma of DIP array
-  const positionGamma = greeks.getGamma(
-    fairValue,
-    dip.strike,
-    yearsUntilMaturity,
-    impliedVol,
-    rfRate
-  )
+  let dipTotalGamma = getDIPGamma(dipArray, fairValue);
```

Comment:
const

Reply:
dipTotalGamma needs to update each time an event triggers scalperPerp(). It will be able to overwrite this declared as a constant?

Reply:
const mean constant in this scope. when the function finishes, it wipes it out and it can be const the next time through